internal/version/last_known_compatible_envoy.txt

```diff
@@ -0,0 +1 @@
+1.19
```

Comment:
let's derive this from last_known so there's less maintenance. Remember, the last_known is only used in docs. It should never be thought of as the actual latestVersion. So, what I mean is if this is for purpose of docs, stripping the minor from last_known_envoy should do the trick!

Reply:
Yes, sure thing!